uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-widget.scss

```diff
@@ -62,8 +62,17 @@
       color: $gray-darker;
     }
 
+    .leave-status-admin_approved,
```

Comment:
i wonder if you even need this classes, you could use bootstrap-classes directly https://getbootstrap.com/docs/3.3/css/#helper-classes-colors

Reply:
That's what I was trying to do before, but how can I map a Bootstrap class name to a leave request status if not with an object mapping in the controller?

Reply:
in the view, with `ng-class`